code/game/turfs/flooring/flooring_decals.dm

```diff
@@ -21,7 +21,7 @@ var/list/floor_decals = list()
 		var/cache_key = "[alpha]-[color]-[dir]-[icon_state]-[layer]"
 		if(!floor_decals[cache_key])
 			var/image/I = image(icon = src.icon, icon_state = src.icon_state, dir = src.dir)
-			I.layer = T.layer
+			I.layer = FLOAT_LAYER
```

Comment:
prob. don't need to set this explicitly, since this is the default layer on images.

Reply:
It can't hurt!

Reply:
True.